dashboard/app/controllers/featured_projects_controller.rb

```diff
@@ -68,4 +76,24 @@ def buffer_abuse_score(score = -50)
       'rack.input' => StringIO.new
     )
   end
+
+  def freeze_featured_project(project_id)
+    project = Project.find_by(id: project_id)
+    return if project.nil?
```

Comment:
I think it's worth a comment here that this early return is here to avoid issues we have with creating projects in unit tests. A little torn on this approach because if we actually were unable to find the project, we'd return from the controller action a success status, without actually freezing the project.

But, if you tried the other path we considered (creating a project in a `setup_all` block, which we've observed working in other places) and that's not working, I get that you have much of a choice.

Reply:
Thanks Ben! I added a comment here as you suggested.
I'll merge as-is with the early return since we'd like the ability to freeze/unfreeze with the already merged updated public gallery work. But I'm going to continue investigating tests in this space and I think I can do a quick follow-up here to address your concern.